src/EnergyPlus/FluidProperties.cc

```diff
@@ -2039,9 +2037,8 @@ namespace FluidProperties {
         // for the refrigerant properties.
         // Most properties requested (e.g., Specific Heat) must be > 0 but the tables may
         // be set up for symmetry and not be limited to just valid values.
-        auto &df = state.dataFluidProps;
```

Comment:
Why?

Reply:
I thought we were not supposed to reference a class? Maybe this is different. I can easily put these back.

Reply:
You can reference a class/struct, what you shouldn't do is reference an array.